tests/e2e/cucumber/features/smoke/admin-settings/users.ocis.feature

```diff
@@ -17,6 +17,14 @@ Feature: spaces management
     And "Admin" logs out
     Then "Alice" logs in
     And "Alice" logs out
+    And "Admin" logs in
+    And "Admin" opens the "admin-settings" app
+    And "Admin" navigates to the users management page
+    And "Admin" creates the following users using users management page
+      | name    | fullname    | email                   | password |
+      | Tester  | Max Testing | maxtesting@owncloud.com | 12345678 |
+    And "Admin" navigates to the users management page
+    And "Admin" deletes the user "maxtesting@owncloud" using users management page
```

Comment:
Is this step really successful?! Because the email address doesn't match the one you provided when creating the user (`.com` is missing).

Reply:
I too have done the "delete user" test and you as reviewer see #8619
next time, please assign it to yourself to avoid duplicating work

<img width="825" alt="image" src="https://user-images.githubusercontent.com/84779829/225834905-090b6256-4991-42b1-9565-023cd27ceee0.png">